account_move_batch_validate/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Account Move Batch Validate
```

Comment:
as this migration is cheap: pls move to new readme ;)
https://github.com/OCA/maintainer-tools/tree/master/template/module/readme